e2e/util/logger.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func CreateLogger(executableName string, verbose bool) *zap.SugaredLogger {
+	logFileName := fmt.Sprintf("%s.log", executableName)
+	logFilePath := filepath.Join(logFileName)
+	consoleConfig := zap.NewProductionEncoderConfig()
+	consoleConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	consoleConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	consoleConfig.CallerKey = zapcore.OmitKey
+	logfileConfig := zap.NewProductionEncoderConfig()
+	logfileConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	logfileConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	consoleEncoder := zapcore.NewConsoleEncoder(consoleConfig)
+	logfileEncoder := zapcore.NewConsoleEncoder(logfileConfig)
```

Comment:
Lets split the code to 2 groups separated with blank line:
- creating console config and encoder
- creating logfile config and encoder